client/src/components/body/lab2/reading.js

```diff
@@ -0,0 +1,151 @@
+import React, {useEffect} from "react";
+import {LAB_ID} from '../../../constants/lab2';
+import UserLabService from '../../../services/UserLabService';
+import { Pie } from "react-chartjs-2";
+
+const data = {
+    labels: [
+      "Males With A CVD",
+      "Females With A CVD",
+      "Males Without A CVD",
+      "Females Without A CVD"
+    ],
+    datasets: [
+      {
+        label: "Color Visions Deficiencies in a Population of 10,000 People",
+        borderColor: "black",
+        backgroundColor: ["red", "orange", "blue", "purple"],
+        data: [416.67, 25, 4583.33, 4975],
+        borderWidth: "2"
+      }
+    ]
+  };
+
+const Reading = () => {
+    useEffect(() => {
+        return () => {
+            UserLabService.complete_reading(LAB_ID);
+        }
+    });
+    return (
+        <div className="study">
+        <h3 >Color Vision Deficiencies</h3>
+        <div className="flex">
+          <Pie data={data} height={100} />
```

Comment:
The "Females with a CVD" sliver of the pie chart is very small and it's hard tell it's even there unless you look very closely - is there a way to make the pie chart bigger?

Reply:
I agree, if a bigger piechart isn't possible, maybe choosing a more prominent color for it (eg. red), and sandwiching it between lighter colors might work?